gauth.go

```diff
@@ -17,13 +19,16 @@ import (
 )
 
 func main() {
+	useCSV := flag.Bool("csv", false, "Output CSV for easy machine processing")
+	flag.Parse()
+
 	cfgPath := os.Getenv("GAUTH_CONFIG")
 	if cfgPath == "" {
-		user, err := user.Current()
+		usr, err := user.Current()
```

Comment:
Just a minor cosmetic thing: If you're going to bother abbreviating anyway, might as well just use `u`. It doesn't escape this scope anyway.